### ISSUE TITLE
The following classes needed for analysis were missing

### ISSUE THREAD

**Issue by delanym** [find-sec-bugs/find-sec-bugs#701]
Where is the fault? https://github.com/spotbugs/spotbugs/issues/527#issuecomment-1508008828

**Comment by bdellegrazie** [comment]
Even with 1.13.0 and taint configuration enabled `'-Dfindsecbugs.taint.workaroundvisitinvokedynamic=true'` when invoked via SpotBugs I'm still seeing the full list (+ others) of lambda methods:
```
  test
  accept
  makeConcatWithConstants
  apply
  hashCode
  equals
  toString
  exposure
  compare
  run
  applyAsLong
  get
  applyAsInt
  isOpen
  onChange
  bootstrap
  barrier
```
I can't guarantee this list is solely the result of Lambdas but _without_ find-bugs-sec added as a SpotBugs plugin I do not see _any_ missing classes

**Comment by pavelorehov** [comment]
Looks like I have similar issue, wonder is issue is the same and if that prevents findsecbugs-plugin from executing any scan ?
Java 11, findsecbugs-plugin 1.13.0

[INFO] --- spotbugs-maven-plugin:4.8.3.1:spotbugs (spotbugs) @ my-service ---
[INFO] Fork Value is true
     [java] The following classes needed for analysis were missing:
     [java]   run
     [java]   makeConcatWithConstants
     [java]   toPredicate
[INFO] Done SpotBugs Analysis....

**Comment by kohlschuetter** [comment]
Likely dup of https://github.com/find-sec-bugs/find-sec-bugs/issues/332

Should be fixed with https://github.com/find-sec-bugs/find-sec-bugs/pull/750

**Comment by Vest** [comment]
The merge was performed 3 weeks ago, but the last release was published in 2024. I’d better wait, till the new version is released.

**Comment by kohlschuetter** [comment]
@Vest Yes, it makes sense to keep this bug open, as it has not been properly addressed in a release (or at least assigned to a future release).

**Comment by bdellegrazie** [comment]
Any chance we can get a release soon please?

**Comment by gtoison** [comment]
Version [1.14.0](https://mvnrepository.com/artifact/com.h3xstream.findsecbugs/findsecbugs-plugin/1.14.0) was release silently over the week-end 😉

**Comment by kohlschuetter** [comment]
This appears fixed with 1.14.0.
